clients/runtime/src/types.rs

```diff
@@ -168,13 +171,11 @@ pub trait PrettyPrint {
 }
 
 mod account_id {
-	use subxt::ext::sp_core::crypto::Ss58Codec;
-
 	use super::*;
 
 	impl PrettyPrint for AccountId {
 		fn pretty_print(&self) -> String {
-			self.to_ss58check_with_version(SS58_PREFIX.into())
+			"self.into()".to_string()
```

Comment:
Is this a missing TODO?

Reply:
It probably was 😅, or a very "convenient" typo. Let me see if the original implementation compiles.